Dockerfile

```diff
@@ -1,7 +1,19 @@
-FROM golang
+FROM golang as builder
```

Comment:
This Dockerfile is used for building `lokoctl`. Converting it to the runtime image with controller binary seems like a slight abuse here. Do we want to have all built binaries in the single image?

Reply:
I think we should have a seperate docker file for it